docs/templates/agent-metadata.md

```diff
@@ -68,6 +66,17 @@ resource "coder_agent" "main" {
 
 ## Utilities
 
+[top](https://linux.die.net/man/1/top) is available in most Linux
+distributions and provides virtual memory, CPU and IO statistics. Running `top`
```

Comment:
```suggestion
distributions and provides virtual memory, CPU and IO statistics. Running `top  -bn1`
```

we'll want to recommend `top -bn1` here because regular top opens a live-refreshing screen view, which is not friendly for parsing.